diffusion/models/stable_diffusion.py

```diff
@@ -342,8 +342,8 @@ def generate(
         seed: Optional[int] = None,
         progress_bar: Optional[bool] = True,
         zero_out_negative_prompt: bool = True,
-        crop_params: Optional[list] = None,
-        size_params: Optional[list] = None,
+        crop_params: Optional[Union[tuple, torch.Tensor]] = None,
```

Comment:
Tuple should of what typing / what size?

Reply:
done!